### ISSUE TITLE
Fake Gopher violates Gopher design rules

### ISSUE THREAD

**Issue by rawtaz** [containous/traefik#3114]
### Do you want to request a *feature* or report a *bug*?
Report a problem arguably more serious than a bug.

### What did you do?
Looked at the Gopher image in the README at https://github.com/containous/traefik, more specifically [this one](https://github.com/containous/traefik/blob/master/docs/img/traefik.logo.png).

### What did you expect to see?
A real Gopher.

### What did you see instead?
A fake Gopher.

### Output of `traefik version`: (_What version of Traefik are you using?_)
N/A

### What is your environment & configuration (arguments, toml, provider, platform, ...)?
N/A

### If applicable, please paste the log output at DEBUG level (`--logLevel=DEBUG` switch)
N/A

### The problem
The purported Gopher in the referenced image is in violation of the de facto official Go Gopher model sheet included in [this presentation by Renee French](https://github.com/gophercon/2016-talks/blob/master/ReneeFrench-TheGoGopherACharacterStudy/gopherconfinal.pdf).

While real Go Gophers have been seen having something resembling two "toes" on their nubs, I am not aware of any occasion where a true Go Gopher has three toes/fingers on their nubs.

The particular "Gopher" in the referenced image is clearly holding his two sticks using not nubs, but hands or something similar thereof. It's quite clear that there's some kind of "thumbs" involved, alongside two "fingers" on each "hand". I suspect these are badly designed gloves, in an attempt to cover up what are not real Gopher nubs.

In the presentation by Renee French, she explains that there are two ways the Gopher can hold things/objects; "under the arm nub, and by touch". I think this is an established fact by now, and it also holds true with any other Gopher image by Renee that I've seen. Please see the page 26-27 in the PDF for the model sheet, and page 32-36 for specifics.

I argue that in addition to the aforementioned pictorial evidence of this not being a true Gopher, the fact that if it *was* a true Gopher he would have decided to hold the sticks either under his arm nubs or, more likely, using the adhesive touch technique, clearly shows that this is an imposter.

In conclusion, the fake Gopher should be replaced with a real Gopher, using real nubs and proper holding techniques (which only real Gophers can perform).

**Comment by emilevauge** [comment]
The whole legal team is on this ⚖️

**Comment by emilevauge** [comment]
Our legal team found a document that clearly proves that gophers have more fingers than you initially stated:

![gopher fingers](https://user-images.githubusercontent.com/6207234/38258671-5e9dbf7e-3763-11e8-96cd-446667a9e7bd.png)

I think we can safely close that issue now.
